<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>734</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>OptiWin</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-office"/>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="manualButton">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>You can manually change settings to your custom preferences.</string>
       </property>
       <property name="text">
        <string>Manual</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="automaticButton">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>You can set to automatic mode to get the best out of your computer. (recommended)</string>
       </property>
       <property name="text">
        <string>Automatic</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1281</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="powersaverButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>190</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Saver</string>
      </property>
     </widget>
     <widget class="QComboBox" name="displayModeComboBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="consoleLogViewer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>530</y>
        <width>711</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="brightnessSlider">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="processListWidget">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>350</y>
        <width>261</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="listProcessesButton">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>270</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>List Processes</string>
      </property>
     </widget>
     <widget class="QLabel" name="brightnessValueLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="brightnessValueLabel_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Change Brightness
</string>
      </property>
     </widget>
     <widget class="QLabel" name="brightnessValueLabel_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Change Refresh Rate
</string>
      </property>
     </widget>
     <widget class="QLabel" name="refreshRateLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="balancedButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>250</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Balanced</string>
      </property>
     </widget>
     <widget class="QPushButton" name="highPerformanceButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>310</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>High Performance</string>
      </property>
     </widget>
     <widget class="QLabel" name="brightnessValueLabel_4">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>140</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Change Power Plan
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="terminateProcessButton">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>210</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Kill Process</string>
      </property>
     </widget>
     <widget class="QLabel" name="logomanual">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="normalModeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Optimize Battery Life</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="turboModeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Optimize Performance</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="consoleLogViewer_automatic">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>530</y>
        <width>711</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="logoautomatic">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="priorityGroup" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>410</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>230</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Auto-Optimize Mode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dynamicModeButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>280</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMaxBrightness">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMinBrightness">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>440</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel_2">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>220</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Brightness:</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel_3">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>400</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Minimum Brightness:</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel_4">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>140</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Monitoring settings</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMaxBatteryBrightness">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel_5">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Maximum Brightness:</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel_6">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>180</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Plugged-in</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel_7">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>290</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>On Battery</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="darkModeToggleButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1284</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
